spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/log/Slf4JSpanLoggerTest.java

```diff
@@ -102,19 +97,17 @@ public void should_remove_entries_from_mdc_for_null_span() throws Exception {
 		ExtraFieldPropagation.set(this.span.context(), "my-baggage", "my-value");
 		ExtraFieldPropagation.set(this.span.context(), "my-propagation",
 				"my-propagation-value");
-		this.slf4jScopeDecorator.decorateScope(this.span.context(), () -> {
```

Comment:
this test case leaked data.. I rewrote it to not do that, but the assertions make the same intent true